doc/nrf/ug_thread_configuring.rst

```diff
@@ -78,6 +78,18 @@ Default configuration reference
 
 For other optional configuration options, see the following sections.
 
+.. _ug_thread_configuring_eui64:
+
+.. include:: /includes/ieee802154_eui64_conf.txt
+
+  #. Check the EUI-64 value using OpenThread CLI:
```

Comment:
The doc build is failing. :(

Try this:
```suggestion
.. include:: /includes/ieee802154_eui64_conf.txt
..

   #. Check the EUI-64 value using OpenThread CLI:
```

If the numbering starts with 1., just use "5." instead of "#." ...

Reply:
@greg-fer is working into some changes, we already agreed a solution for this.